app/src/main/java/com/keylesspalace/tusky/LoginActivity.java

```diff
@@ -234,15 +253,11 @@ private static String toQueryString(Map<String, String> parameters) {
     }
 
     private static boolean openInCustomTab(Uri uri, Context context) {
-        boolean lightTheme = PreferenceManager.getDefaultSharedPreferences(context)
-                .getBoolean("lightTheme", false);
-        int toolbarColorRes;
-        if (lightTheme) {
-            toolbarColorRes = R.color.custom_tab_toolbar_light;
-        } else {
-            toolbarColorRes = R.color.custom_tab_toolbar_dark;
-        }
-        int toolbarColor = ContextCompat.getColor(context, toolbarColorRes);
+        TypedValue toolbarColorTv = new TypedValue();
+        context.getTheme().resolveAttribute(
+                context.getResources().getIdentifier("custom_tab_toolbar", "attr", context.getPackageName()), toolbarColorTv, true);
+
+        int toolbarColor = ContextCompat.getColor(context, toolbarColorTv.resourceId);
```

Comment:
there should already be a helper method for this in com.keylesspalace.tusky.util.ThemeUtils (if not, add one)